### PR TITLE
Remove svg support

### LINKED ISSUE #668: SplashScreen looks useless
I noticed that there's a few issues with the SplashScreen thing. First, it is visible for so short time that it basically looks like visual glitch. Second, it reuses AboutPanel, but contrary to what name of method used to obtain AboutPanel instance, which is `getImagePanel()` suggests, returned instance includes the whole credits scroller, which is a quite odd thing for a splash screen. But as the it is gone so fast, nobody managed to noticed this so far :) which gets me to the conclusion that maybe the splash screen thingie serves no purpose nowadays and can safely be removed. Am I correct?

### LINKED ISSUE #709: Build Errors in Eclipse on develop branch
Tonight I refreshed my local copy of the devlop branch, did gradlew run which succeeded but when I went to build using Eclipse (imported Gradle project) I got lots of errors (see below). The master branch builds w/o error using the same process, as did the develop branch a few days ago.

2021-06-22 19_53_21-Eclipse-logisim-evolutionMaster - Eclipse IDE

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
* Replaced SVG in-app logo with PNG
* SVG logo (version with fonts and all-curved) moved to `artwork/`
* Added logo related notes to separated README in `artwork/`
* Removed Batik gradle dependency
* Updated *.md to refer to new location of SVG logo
* Marked SVG files as `binary` type, so they are not diff-ed

Closes #709
Closes #668

### CODE REVIEW

File: artwork/README.md [hunk 1]
```diff
@@ -2,9 +2,12 @@
...
 
-Logisim-evolution logo project
+# Logisim-evolution logo project #
+
 Designed by Marcin Orlowski <http://MarcinOrlowski.com>
 
+## SVG files ##
+
 * `logisim-evolution-logo-src.svg` - main logo project file
 * 'logisim-evolution-logo.svg` - derrived from main project file with all texts converted to paths.
```
Comment (maehne) [inline artwork/README.md:12]: Typo: "derrived" -> "derived"

**Review by maehne — APPROVED** [review]
Looks good to me! Thanks @MarcinOrlowski!

File: artwork/README.md (hunk 1 above)
Comment (MarcinOrlowski) [reply artwork/README.md]: Fixed

**Review by davidhutchens — APPROVED** [review]
I confirm that this does work with Eclipse on my Mac.

**Pull request merged**